pkgs/tools/networking/remctl/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, autoreconfHook, fetchgit, libkrb5, libevent, openssl, perl
+, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "remctl";
+  version = "3.17";
```

Comment:
Can we get 3.18 from https://github.com/rra/remctl ? I can't seem to browse their git.